### PR TITLE
SAMZA-1761: Reduce runTime of TestZkUtils test from 40s to 800ms.

### PR DESCRIPTION
**Pull request by shanthoosh**

### CODE REVIEW

**Comment by shanthoosh** [comment]
@xinyuiscool @vjagadish1989 @sborya 
Please take a look when you get a chance.

File: samza-core/src/test/java/org/apache/samza/zk/TestZkUtils.java [hunk 1]
```diff
@@ -265,7 +265,7 @@ public void testRegisterProcessorAndGetIdShouldFailForDuplicateProcessorRegistra
     // Register processor 1 which is not duplicate, this registration should succeed.
     zkUtils.registerProcessorAndGetId(processorData1);
 
-    ZkUtils zkUtils1 = getZkUtils();
+    ZkUtils zkUtils1 = new ZkUtils(KEY_BUILDER, zkClient, 1000, 500, new NoOpMetricsRegistry());
```
Comment (prateekm) [inline samza-core/src/test/java/org/apache/samza/zk/TestZkUtils.java]: Can we reduce the value of SESSION_TIMEOUT_MS and CONNECTION_TIMEOUT_MS constants that getZkUtils uses?

Comment (shanthoosh) [reply samza-core/src/test/java/org/apache/samza/zk/TestZkUtils.java]: Makes sense. Done.

**Comment by vjagadish1989** [comment]
merged, submitted thanks!

**Pull request closed without merging**